fastlane/lanes/build.rb

```diff
@@ -134,6 +134,27 @@
     end
   end
 
+  #####################################################################################
+  # download_signed_apk_from_google_play
+  # -----------------------------------------------------------------------------------
+  # This lane downloads the signed apk from Play Store for the given version and app
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane download_signed_apk_from_google_play app:<wordpress|jetpack> version:<versionName,versionCode>
+  #####################################################################################
+  lane :download_signed_apk_from_google_play do |options|
+    app = get_app_name_option!(options)
+    package_name = APP_SPECIFIC_VALUES[app.to_sym][:package_name]
+    version = options[:version] || android_get_release_version()
```

Comment:
iinm this line would make `version` be a `String` if passed explicitly via `version:xyz` option, while it would be a `Hash` if not passed as an option and we end up calling `android_get_release_version`, no?

Which would make the rest of the code trying to read things like `version[:code]` a couple of lines below crash if provided as an option aka à `String` not à `Hash`?

Reply:
I think [my previous comment](https://github.com/wordpress-mobile/WordPress-Android/pull/18813#discussion_r1271162781) about the documentation covers this. This is just how the version is handled across other lanes, so I am just following our current practice.